lib/main.js

```diff
@@ -70,14 +71,22 @@ export default {
     this.subscriptions.add(atom.commands.add('atom-text-editor', {
       'linter-eslint:fix-file': function() {
         const textEditor = atom.workspace.getActiveTextEditor()
+        const filePath = textEditor.getPath()
+        const fileDir = Path.dirname(filePath)
+        const params = {
+          fileDir,
+          global: atom.config.get('linter-eslint.useGlobalEslint'),
+          nodePath: atom.config.get('linter-eslint.globalNodePath'),
+        }
+        const eslintDir = findEslintDir(params)
```

Comment:
I would highly recommend that we do not do this in the main process, the sync file system operations and sync execution is gonna make the editor jam for the user